packages/cable/package.py

```diff
@@ -17,9 +14,10 @@ class Cable(CMakePackage):
     homepage = "https://github.com/CABLE-LSM/CABLE"
     git = "https://github.com/CABLE-LSM/CABLE.git"
 
-    maintainers("SeanBryan51")
+    maintainers("SeanBryan51", "Whyborn")
 
```

Comment:
```suggestion

license("CSIRO Open Source Software Licence v1.0", checked_by="anton-seaice")

```

There's no SPDX code for the CSIRO license, so maybe listing nothing is better

Reply:
@harshula My reading of this doesn't provide any useful information on licences which don't have a SPDX ID (of which the CSIRO license is one). The [CSIRO License](https://research.csiro.au/dap/licences/csiro-open-source-software-licence-based-on-mit-bsd-open-source-licence/) does say it's based on the MIT/BSD open source license, but is distinct due to some CSIRO clauses.

Reply:
Hi @Whyborn , The first example in the documentation clearly states, `license("<SPDX Identifier HERE>")`. Would you like me to check with Spack upstream what we should do if there is no SPDX Identifier?

Reply:
I swear I searched for this before, but better luck this time!

In https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/

it says use:

```
 	"LicenseRef-"[idstring]
where
[idstring] is a unique string containing letters, numbers, . and/or -.
```

in this case 

`license("LicenseRef-CSIRO-Open-Source-Software-Licence-v1.0", checked_by="anton-seaice")`

might be better?